### PR TITLE
PAE-273: Custom theme for Learning Hub - Style login

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description:
---------------
The current style was updated by eliminating the old

Before:
image

After:
image

### CODE REVIEW

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss [hunk 1]
```diff
@@ -1,46 +1,44 @@
...
-  p, p a, li {
-    color: $secondary;
-  }
-}
-
-
-div.login-register div.toggle-form a.form-toggle {
-  color: $primary;
+.signin-container {
+    box-shadow: 2px 2px 2px #eee;
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: Can you get this color from variables definitions? @luismorenolopera

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss [hunk 2]
```diff
@@ -1,46 +1,44 @@
...
+    font-size: 18px;
+    font-weight: 600;
+}
+.signing-title span {
+    margin-bottom: 10px;
+}
+.login-register .form-wrapper .login-providers {
+    display: flex;
+    text-align: left;
+    background: #f8f8f8;
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: Can you get this color from variables definitions? @luismorenolopera

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss [hunk 3]
```diff
@@ -1,46 +1,44 @@
...
+    padding: 10px 15px 40px;
+    justify-content: space-between;
+    align-items: center;
+    box-shadow: 2px 2px 2px #eee;
+    border-bottom: 0.1px solid #919191;
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: Can you get these colors from variables definitions? @luismorenolopera

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss [hunk 4]
```diff
@@ -1,46 +1,44 @@
...
+}
+.email-email .label-text,
+.password-password .label-text {
+    display: block;
+    margin: 10px 0;
+    height: 0
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: Missing semicolon. @luismorenolopera

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss [hunk 5]
```diff
@@ -1,46 +1,44 @@
...
+    height: 45px;
+    padding: 15px;
+    margin-bottom: 5px;
+}
+.login-register div.toggle-form {
+    margin-top: 15px;
+    text-align: center;
+}
+.login-register .form-field .tip {
+    color: #999;
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: Can you get these colors from variables definitions? @luismorenolopera

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss [hunk 6]
```diff
@@ -1,46 +1,44 @@
...
+.signin-container {
+    box-shadow: 2px 2px 2px #eee;
+    padding: 50px 30px 60px;
+}
+.signing-title {
+    text-align: center;
+    font-size: 18px;
+    font-weight: 600;
+}
+.signing-title span {
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: This definition can be added to the .signing-title block. @luismorenolopera

File: edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+              <% if ( provider.iconImage ) { %>
+              <img class="icon-image" src="<%- provider.iconImage %>" alt="<%- provider.name %> icon" />
+              <% } %>
+          </div>
+          <span aria-hidden="true"><%- provider.name %></span>
+          <span
+              class="sr"><%- _.sprintf( gettext("Sign in with %(providerName)s"), {providerName: provider.name} ) %></span>
+      </button>
+    </div>
+    <% }
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore]: Wrong indentation. @luismorenolopera

File: edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+        <%- gettext("You already have an edX account with your {enterprise_name} email address.").replace(/{enterprise_name}/g, context.enterpriseName) %>
+        <% if (context.syncLearnerProfileData) {
+            %><%- gettext("Going forward, your account information will be updated and maintained by {enterprise_name}.").replace(/{enterprise_name}/g, context.enterpriseName) %>
+        <% } %>
+        <%- gettext("You can view your information or unlink from {enterprise_name} anytime in your Account Settings.").replace(/{enterprise_name}/g, context.enterpriseName) %>
+    </p>
+    <p><%- gettext("To continue learning with this account, sign in below.") %></p>
+    <% } else { %>
+    <span class="signing-title">If you're enrolled on Learning Hub courses, sign in here:</span>
+    <% } %>
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore]: Wrong indentation. @luismorenolopera

File: edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+        <button type="submit"
+            class="action action-primary action-update js-login login-button"><%- gettext("Sign in") %></button>
+
+        <% if ( context.hasSecondaryProviders ) { %>
+        <button type="button" class="button-secondary-login form-toggle" data-type="institution_login">
+            <%- gettext("Use my institution/campus credentials") %>
+        </button>
+        <% } %>
+        <% } %>
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore:69]: What closes this curly bracket? @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore (hunk 9 above)
Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore:69]: line 1

File: edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore (hunk 8 above)
Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore]: done

File: edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore (hunk 7 above)
Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/templates/student_account/login.underscore]: done

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss (hunk 6 above)
Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: ok

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss (hunk 5 above)
Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: yes

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss (hunk 4 above)
Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: done

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss (hunk 3 above)
Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: yes

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss (hunk 2 above)
Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: yes

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss (hunk 1 above)
Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: yes

File: edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss [hunk 10]
```diff
@@ -1,46 +1,49 @@
...
+    font-weight: 600;
+    margin-bottom: 10px;
 }
 
-.login-register {
-  background: $background-color-form-login-register;
+.login-register .form-wrapper .login-providers {
+    display: flex;
+    text-align: left;
+    background: $guyabano-gray;
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: guyabano? it is a color? @luismorenolopera Just asking...

Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: guyabano? it is a color? @luismorenolopera Just asking...

Comment (luismorenolopera) [reply edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: Based on this page yes, https://encycolorpedia.com/f8f8f8

Comment (Squirrel18) [reply edx-platform/pearson-learninghub-theme/lms/static/sass/partials/lms/theme/_login-register.scss]: Haha really weird, but...

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**